cttportfolio.html

```diff
@@ -0,0 +1,106 @@
+<!doctype html>
+<html lang= "en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="UTF-8">
+	<title>HTML Exercise 1</title>
+	<link rel="stylesheet" type="text/css" href="cttportfolio.css">
+</head>
+<body>
+	<nav>
+		<a href="cttportfolio.html"><img src="https://placeholder.com/wp-content/uploads/2018/10/placeholder.com-logo3.png" alt="Logo replacement"></a> 
```

Comment:
Personal preference here, but I think it would be easier to read if you formatted this line like:
```html
<a>
    <img>
</a>
```